### ISSUE TITLE
running without metamask

### ISSUE THREAD

**Issue by rzwnk** [oraclesorg/ico-wizard#359]
Can we run ico wizard without metamask by tweaking the getWeb3 method, using unlocked geth account?

**Comment by igorbarinov** [comment]
Probably you can. What is your use case?

Some remarks:

- We tried with parity trusted signer and it worked.
- For invest page there is a PR to make contribution without metamask https://github.com/oraclesorg/ico-wizard/pull/199

**Comment by igorbarinov** [comment]
@rzwnk please let me know

**Comment by rzwnk** [comment]
Thanks @igorbarinov  for your response, I have a local geth node running, I removed the window.web3 code in getWeb3 method, after doing this was able to run ico wizard with default account in geth but I could not complete crowdsale as crowdsale address was not being sent to crowdsale page, that is why I wanted to ask if the code is written to be completely dependent on metamask. I had done this a few days ago but yesterday I took a pull again and it changed several files, after doing this I was able to successfully complete crowdsale on ropsten testnet. 
I will close this issue now.
